apps/api/src/settings/constants/settings.constants.ts

```diff
@@ -6,3 +6,11 @@ export const DEFAULT_USER_ADMIN_SETTINGS = {
   ...DEFAULT_USER_SETTINGS,
   adminNewUserNotification: false,
 };
+
+export const DEFAULT_GLOBAL_SETTINGS = {
+  platformLogoS3Key: null as string | null,
+} as const;
+
+export type GlobalSettingsType = {
```

Comment:
Types should be kept in types file.

Reply:
It's not needed to have `type` word in it, and also i think it should consist `JSON` in name, since it is JSON